packages/common/src/allo/roundIdentifier.ts

```diff
@@ -0,0 +1,45 @@
+export type RoundType =
+  | "MERKLE"
+  | "DIRECT"
+  | "SUPERFLUID"
+  | "MICROGRANTS"
+  | "MICROGRANTSGOV"
+  | "MICROGRANTSHATS"
+  | "RFPSIMPLE"
+  | "RFPCOMMITTEE"
+  | "QVSIMPLE";
+
+export const getV2RoundType = (poolId: string): RoundType => {
```

Comment:
not sure if it helps but in the indexer we have the strategyName, including allo v1 qf and direct

Reply:
just want to make sure we won't have to repeat this code (for v2 and v1) if it's already in the indexer

Reply:
Good call out, this is implemented as a short term fix.